### PR TITLE
CoAp: Addressing review findings

### PR DESCRIPTION
**Pull request by husigeza**
Fixes for the review findings here: https://github.com/pycom/pycom-documentation/commit/e2eca78b3991b21608cab2e6748b6e6dbf74d3fa

### CODE REVIEW

File: content/firmwareapi/pycom/network/coap.md [hunk 1]
```diff
@@ -34,17 +34,17 @@ def socket_thread(p, coap_socket):
             sock = s[0]
             event = s[1]
             if(event & uselect.POLLIN):
-                # Check if the socket belongs to Coap module
+                # Check if the socket belongs to CoAp module
```
Comment (lilycey) [inline content/firmwareapi/pycom/network/coap.md]: First comment: Check if the socket belongs to the CoAp module

File: content/firmwareapi/pycom/network/coap.md [hunk 2]
```diff
@@ -34,17 +34,17 @@ def socket_thread(p, coap_socket):
...
+                    # Call Coap.read() which parses the incoming CoAp message
                     Coap.read()
 
 
 # Connect to the network
 wlan = WLAN(mode=WLAN.STA)
 wlan.connect('your-ssid', auth=(WLAN.WPA2, 'your-key'))
 
-# Initialize Coap module
+# Initialize CoAp module
```
Comment (lilycey) [inline content/firmwareapi/pycom/network/coap.md]: Second comment: # Initialise the CoAp module

File: content/firmwareapi/pycom/network/coap.md [hunk 3]
```diff
@@ -34,17 +34,17 @@ def socket_thread(p, coap_socket):
...
 Coap.init(str(wlan.ifconfig()[0]), service_discovery=True)
 
 # Add a resource with default value and plain text content format
```
Comment (lilycey) [inline content/firmwareapi/pycom/network/coap.md]: Third: Add a resource with a default value and a plain text content format

**Review by lilycey — COMMENTED** [review]
Only three tiny changes

File: content/firmwareapi/pycom/network/coap.md (hunk 1 above)
Comment (husigeza) [reply content/firmwareapi/pycom/network/coap.md]: Fixed.

File: content/firmwareapi/pycom/network/coap.md (hunk 3 above)
Comment (husigeza) [reply content/firmwareapi/pycom/network/coap.md]: Fixed.

File: content/firmwareapi/pycom/network/coap.md (hunk 2 above)
Comment (husigeza) [reply content/firmwareapi/pycom/network/coap.md]: Fixed.

**Review by lilycey — APPROVED** [review]
All looks good to me!

**Pull request merged**